websockify/websocketproxy.py

```diff
@@ -183,6 +183,11 @@ def do_proxy(self, target):
         else:
             self.heartbeat = None
 
+        # see comment below - an assertion to deal with the
+        # intrinsics of non-blocking SSL sockets and select.
+        assert isinstance(target, ssl.SSLSocket) \
+               and self.buffer_size >= 16 * 1024 and not target.getblocking()
```

Comment:
This assert seems wrong. We don't want to remove support for non-SSL sockets.

Reply:
Yep, it is. Sorry, I don't know how I missed that 😅

Reply:
That should be fixed now 👍